libdnf/libdnf.pc.in

```diff
@@ -5,7 +5,6 @@ includedir=${prefix}/include
 Name: libdnf
 Description: Simple package manager interface using libsolv and librepo
 Version: @LIBDNF_VERSION@
-Requires: glib-2.0, librepo, rpm, sqlite3
-Requires.private: libsolv, libsolvext
+Requires: glib-2.0, librepo >= 0.11, rpm, libsolv, libsolvext
```

Comment:
The `librepo` version is wrong.

Reply:
Requires.private is so static linking so yes if you want to support static linking with libdnf sqlite3 should be moved to Requires.private but definitely should not be listed in Requires :)